modules/ui/edit_menu.js

```diff
@@ -126,7 +126,7 @@ export function uiEditMenu(context) {
         }
 
         // update
-        buttons = buttonsEnter
+        buttonsEnter
```

Comment:
We actually have to do the assignment when we merge, or else the results are discarded. See https://github.com/d3/d3-selection/blob/v2.0.0/README.md#selection_merge. Could you change this back?

Reply:
Sure

Reply:
I might be wrong, but I think that the assignment isn't used (it's not captured in a closure, or used later in the program).

Reply:
Oh okay, looks like you're right.